hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -623,20 +689,29 @@ private synchronized SnapshotDiffJob getSnapDiffReportStatus(
     return snapDiffJob;
   }
 
-  private void validateSnapshotsAreActive(final String volumeName,
-                                          final String bucketName,
-                                          final String fromSnapshotName,
-                                          final String toSnapshotName)
+  private boolean validateDiffJobAndSnapshotsAreActive(
```

Comment:
```suggestion
  private boolean areDiffJobAndSnapshotsActive(
```

since it returns `boolean` now